dist/index.html

```diff
@@ -568,7 +568,7 @@ <h2>Kluster</h2>
               java -cp h2odriver-extended.jar water.H2OApp -md5skip -name test
             </p>
 
-            <p>4 In your Sparkling Water application, create H2OContext:</p>
+            <p>4. In your Sparkling Water application, create H2OContext:</p>
```

Comment:
should not be `4.` but `5.`

Reply:
? ... I still only get to step 4:
https://github.com/h2oai/sparkling-water/blob/b553ee23cbe62e6596976e0a77a3708497e97d37/dist/index.html#L560

Reply:
Yup, this was cased by the fact Github shrimped the page and I saw two 4. right after each other :D